packages/ember-data/tests/unit/model/errors_test.js

```diff
@@ -30,7 +30,6 @@ function unexpectedSend(eventName) {
 }
 
 test("add error", function() {
-  expect(6);
```

Comment:
Synchronous expects make the test

* noisy - if an expectation fails, the subsequent failures won't be run anyway
* brittle - adding/removing an additional expectation requires both adding/removing the expectation, as well as incrementing/decrementing `expect(n)`